apps/client/src/pricers/defillama/index.ts

```
@@ -48,9 +35,6 @@ export class DefiLlamaPricer implements Pricer {
     }
 
     const price = await this.fetchPrice(client, asset);
-    if (price === undefined) {
-      throw new Error(`No price found for asset ${asset} on ${client.chain.name}`);
-    }
```

Didn't catch this when I was reviewing you PR but I don't think we should throw an error when the price fetching fails

---

Fair, but then I think the fetch should be wrapped in a `try` `catch` that returns undefined on any error.

---

It's already the case right ?